app_widgetbook/lib/core/ui_elements/app_bottom_navigation_bar.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:core/core.dart';
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart' show Knobs;
+import 'package:widgetbook_annotation/widgetbook_annotation.dart';
+
+@WidgetbookUseCase(name: 'Default', type: AppBottomNavigationBar)
+Widget appBottomNavigationBarDefaultUseCase(BuildContext context) {
+  return Align(
+    alignment: Alignment.bottomCenter,
+    child: AppBottomNavigationBar(
+      currentIndex: context.knobs
+          .slider(
+            label: 'Active Tab',
+            initialValue: 0,
+            min: 0,
+            max: 3,
+            divisions: 3,
+          )
+          .toInt(),
+      onItemChanged: (_) {},
+    ),
+  );
+}
```

Comment:
Having the active tab as a slider is kind of funny, but works 😀

Reply:
Yeah I thought it was more accessible than a dropdown, but can change it later. Also it might be temporary if I change some logic for when routing is implemented.